### PR TITLE
feat: [PIE-5210]: Changes for Jira Update

### PR DESCRIPTION
**Pull request by vkbansal-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

## [Contributor license agreement](https://github.com/harness/harness-core-ui/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 5 Issues 
* Code issue 5 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11131)

File: src/modules/70-pipeline/components/InputSetView/TextFieldInputSetView/TextFieldInputSetView.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ interface TextFieldInputSetViewProps {
   name: string
   label: string | ReactElement
   fieldPath: string
+  className?: string
```
Comment (ishantmehtaharness) [inline src/modules/70-pipeline/components/InputSetView/TextFieldInputSetView/TextFieldInputSetView.tsx:19]: @rameshx has already added support for className.

Comment (ishantmehtaharness) [reply src/modules/70-pipeline/components/InputSetView/TextFieldInputSetView/TextFieldInputSetView.tsx:19]: line 27. its there

Comment (vkbansal-harness) [reply src/modules/70-pipeline/components/InputSetView/TextFieldInputSetView/TextFieldInputSetView.tsx:19]: I don't see any conflicts

Comment (ishantmehtaharness) [reply src/modules/70-pipeline/components/InputSetView/TextFieldInputSetView/TextFieldInputSetView.tsx:19]: There is no conflict. There is duplicacy
`TextFieldInputSetViewProps` - here className is defined twice.
and in `<FormInput.MultiTextInput`, its passed twice

File: src/modules/70-pipeline/components/InputSetView/TextFieldInputSetView/TextFieldInputSetView.tsx [hunk 2]
```diff
@@ -51,8 +51,7 @@ export function TextFieldInputSetView(props: TextFieldInputSetViewProps): JSX.El
     allowedTypes: defaultTo(multiTextInputProps?.allowableTypes, [MultiTypeInputType.FIXED]),
     template: template,
     readonly: readonly,
-    tooltipProps: tooltipProps,
-    className
```
Comment (ishantmehtaharness) [inline src/modules/70-pipeline/components/InputSetView/TextFieldInputSetView/TextFieldInputSetView.tsx:55]: We need it here

Comment (vkbansal-harness) [reply src/modules/70-pipeline/components/InputSetView/TextFieldInputSetView/TextFieldInputSetView.tsx]: fixed

**Review by ishantmehtaharness — APPROVED** [review]

**Pull request merged**